pygmt/tests/test_grdimage_image.py

```diff
@@ -3,7 +3,8 @@
 """
 
 import pytest
-from pygmt import Figure, which
+from pygmt import Figure
+from pygmt.datasets import load_blue_marble
 
 rioxarray = pytest.importorskip("rioxarray")
```

Comment:
Can this line be removed?
```suggestion
```

Reply:
Not yet, because we are still using `tempfile_from_image` to convert `xarray.DataArray` to a GeoTIFF before passing into `grdimage`. But see #3468.